cmd/syncer/main.go

```diff
@@ -189,6 +189,20 @@ func initSyncerComponents(ctx context.Context, clusterFlavor cnstypes.CnsCluster
 				}
 			}()
 		}
+		if clusterFlavor == cnstypes.CnsClusterFlavorVanilla {
```

Comment:
why did we move this logic from pkg/syncer/cnsoperator/manager/init.go to here in the main?

Reply:
Node manager needs to be initialized for the CSINodeTopology CR informer to work in `InitMetadataSyncer`. Earlier, the initialization was happening inside the goroutine in `InitCnsOperator` function. Without this change, we cannot guarantee nodes are initialized before the logic runs in `InitMetadataSyncer`